.github/workflows/CompatHelper.yml

```diff
@@ -6,11 +6,14 @@ on:
 jobs:
   CompatHelper:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        subdir: ["", "docs", "test"]
     steps:
       - name: Pkg.add("CompatHelper")
         run: julia -e 'using Pkg; Pkg.add("CompatHelper")'
       - name: CompatHelper.main()
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}
-        run: julia -e 'using CompatHelper; CompatHelper.main()'
+        run: julia -e 'using CompatHelper; CompatHelper.main(; subdirs=["${{ matrix.subdir }}"])'
```

Comment:
For subdirs maybe set to only keep new and remove old compat? Test and docs don't have to be so flexible in terms of their compat entries

Reply:
Like in a802653 ?

Reply:
Seems that this breaks CI. Any suggestions?